### DISCUSSION TITLE
Change the way vdesk handle the process/window asosiation

### DISCUSSION THREAD

**Discussion by LittleVaaty** [LittleVaaty/VDesk#29] [General]
Currently a window is retrieve by the name of the process. It work fine for some process but with other app it doesn't.
I need to change the way process are handle, but it will take some time as i need to read a lot of documentation.

This issue will track all the bug related to this.
- #26 
- #24 
-  #25 
- #16 
- #9 
- #35

**Comment by Dave-1337** [comment]
Hello, have you perhaps found anything about the problem?

**Comment by LittleVaaty** [comment]
Hi, I know I've been quiet lately, but the development is complete.

**Comment by LittleVaaty** [comment]
Hello,
This is still not working. I'm encountering issues with the Windows API calls and memory management.
UWP apps are not meant to be accessed using tools like VDesk.
I'm currently working on an alternative approach, but it will take some time.

Thanks for your patience.

**Comment by Dave-1337** [comment]
Thank you for not giving up :)

**Comment by LittleVaaty** [comment]
It took me some time, but beta 8 is finally here! I hope it will help me investigate issues — both old and new — more effectively.
Don’t expect beta 8 to be fully stable yet, as I still need to find a way to test it automatically, but I’m working on it.
